engine/time/build.gradle

```diff
@@ -27,3 +27,9 @@ dependencies {
             project(path: ':test-configs')
     Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
 }
+
+test {
+    jvmArgs '--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED'
+    systemProperty 'RealTimeClockLoader.timeSource', 'jdk-internals'
+//    systemProperty 'RealTimeClockLoader.timeSource', 'millis'
```

Comment:
Ideally one would test both ways, though the millis and instant implementations are trivial.  You might add a comment to that effect.